### DISCUSSION TITLE
Avoid console of springboot to be overwritten

### DISCUSSION THREAD

**Discussion by Thaybow** [cucumber/common#2182] [Q&A Java]
Running cucumber tests in springboot hide console of springboot . Is it  possible to change this behavior ?

**Comment by mpkorstanje** [comment]
I don't really understand your question. You'll want to read these:
- https://stackoverflow.com/help/how-to-ask
- https://stackoverflow.com/help/minimal-reproducible-example

It takes work and effort to ask a good question. But that is the price of asking random strangers online.

**Reply by Thaybow** [reply]
I mean, when you run a springboot app, you see the banner at start, stackerrors, all your sysout, logs etc etc etc. But when running cucumber, you only see those logs :
 Running FAT Tests
@UC4301, @UCA11,Scenario: Foo                          # features/foo.feature:4
  Given bar

**Reply by mpkorstanje** [reply]
That's not something Cucumber does out of the box. How would I reproduce your problem with the least amount of code?
